feat: Add toast notification for Contact Us form

nayakOPS opened the pull request:
This PR adds a toast notification to the "Contact Us" form to provide feedback to users when their message is successfully sent.

## Changes
- Installed `react-toastify` for toast notifications.
- Added a success toast with a green background and the message: "Your message has been sent successfully."
- The toast appears at the top of the screen and disappears after 3 seconds.

## Testing
- Submitted the form and verified that the toast appears as expected.
- 
Fixes #48

nischalshakya787 commented on src/app/components/ContactUs.tsx:
```diff
@@ -42,8 +43,28 @@ export default function ContactUs() {
       });
       reset();
+
+      toast.success("Your message has been sent successfully!", {
+        position: "top-center",
+        autoClose: 3000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
```
Since the ToastContainer configuration (position, autoClose, etc.) is already defined in layout file, you can just call toast.success("Message") – it will automatically use those settings. No need to repeat the same rules every time. 😊
```suggestion
      toast.success("Your message has been sent successfully!");
```

nischalshakya787 commented on src/app/components/ContactUs.tsx:
```diff
@@ -42,8 +43,28 @@ export default function ContactUs() {
     } catch (error) {
       console.log(error);
+
+      toast.error("Failed to send message. Please try again.", {
+        position: "top-center",
+        autoClose: 3000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
```
```suggestion
      toast.error("Failed to send message. Please try again.");
```

nischalshakya787 commented on src/app/layout.tsx:
```diff
@@ -31,6 +33,17 @@ export default async function RootLayout({
         <Providers>
           <StarField/>
           <Box sx={{ position: "relative", zIndex: 1 }}>{children}</Box>
+            <ToastContainer
+              position="top-center"
+              autoClose={3000}
+              hideProgressBar={false}
+              newestOnTop={false}
+              closeOnClick
+              rtl={false}
+              pauseOnFocusLoss
+              draggable
+              pauseOnHover
+            />
```
And i think it will look good if the toast is at top-right of the window. And you can increase the width or decrease the font-size to make the description fit in one line 
![Screenshot from 2025-01-24 19-03-20](https://github.com/user-attachments/assets/525b8f84-1068-4ab6-b99c-06104a361b72)

nischalshakya787 reviewed (commented):
Overall it looks good. Added few suggestions

nayakOPS commented:
@nischalshakya787 I've addressed your feedback:
- Removed redundant toast configuration in `ContactUs.tsx`.
- Moved the toast to the top-right of the screen.

Please review the changes and let me know if anything else is needed. 😊

Moved the toast Message to top-right:
![Updated UI V2](https://github.com/user-attachments/assets/d667646c-2d9d-47bb-8c36-2448d982ada1)

nayakOPS commented:
![Screenshot from 2025-01-26 11-56-46](https://github.com/user-attachments/assets/c51ca319-f447-499e-9a64-10021b0f207c)

nayakOPS commented:
![Screenshot from 2025-01-26 11-56-46](https://github.com/user-attachments/assets/c51ca319-f447-499e-9a64-10021b0f207c)

bhimrazy reviewed (approved):
LGTM
